getting-started/installation-windows-plain.rst

```diff
@@ -0,0 +1,278 @@
+    Copyright (C) Microsoft Corporation.  All rights reserved.
+
+Change into the testioc folder and run ``make`` (or ``gmake`` when using the make from Strawberry Perl)::
+
+    >cd %HOMEPATH%\testioc
+    >make
+
+This should build the executable and create all files for the test IOC::
+    
+    >dir /b
+    bin
+    configure
+    db
+    dbd
+    iocBoot
+    lib
+    Makefile
+    testApp
+
+At this point, you can run the IOC from either an MSYS2 Bash shell or from a Windows command prompt, by changing into the IOC directory and running the test.exe binary with your startup command script as parameter.
+
+In the Windows ``command prompt``::
+
+    >cd %HOMEPATH%\testioc\iocBoot\ioctest    
+    >..\..\bin\windows-x64\test st.cmd
+
+Or - if you have an installation - in the MSYS2 shell::
+
+    $ cd ~/testioc/iocBoot/ioctest    
+    $ ../../bin/windows-x64/test st.cmd
+
+
+In both cases, the IOC should start like this::
+
+    Starting iocInit
+    #!../../bin/windows-x64/test
+    < envPaths
+    epicsEnvSet("IOC","ioctest")
+    epicsEnvSet("TOP","C:/Users/'user'/testioc")
+    epicsEnvSet("EPICS_BASE","C:/Users/'user'/base-R7.0.4.1")
+    cd "C:/Users/'user'/testioc"
+    ## Register all support components
+    dbLoadDatabase "dbd/test.dbd"
+    test_registerRecordDeviceDriver pdbbase
+    ## Load record instances
+    dbLoadRecords("db/test.db","user='user'")
+    cd "C:/Users/'user'/testioc/iocBoot/ioctest"
+    iocInit
+    ############################################################################
+    ## EPICS R7.0.4.1
+    ## Rev. 2020-10-21T11:57+0200
+    ############################################################################
+    iocRun: All initialization complete
+    ## Start any sequence programs
+    #seq sncxxx,"user='user'"
+    epics>
+
+Check if the database ``test.db`` you created is loaded correctly::
+
+    epics> dbl
+    test:pv1
+    test:pv2
+    test:add
+
+As you can see 3 process variable is loaded and available. Keep this terminal open and running. Test this process variable using another terminals.
+
+Open another shell for monitoring ``test:add``::
+
+    >camonitor test:add
+    test:add                       2020-10-23 13:39:14.795006 100
+
+That terminal will monitor the PV ``test:add`` continuously. If any value change is detected, it will be updated in this terminal. Keep it open to observe the behaviour.
+
+Open a third shell. Using caput, modify the values of  ``test:pv1`` and ``test:pv2`` as we have done in the temperature example above. You will see changes of their sum in the second terminal accordingly.
+
+At this point, you have one IOC ``testioc`` running, which loaded the database ``test.db`` with 3 records. From other processes, you can connect to these records using Channel Access. If you add more process variable in ``test.db``, you will have to ``make`` the `testioc` application again and restart the IOC to load the new version of the database.
```

Comment:
you will not have to ``make`` the `testioc` application, just restart the IOC to load the new version of the database.

Reply:
You have to re-run make, else the database files (in `...App/Db`) will not get installed (in `TOP/db`) where the IOC loads them from.